packages/@uppy/core/src/index.js

```diff
@@ -615,6 +621,7 @@ class Uppy {
 
   reset () {
     this.cancelAll()
+    this.setState({ allowNewUpload: true })
```

Comment:
When I press “Cancel” in Dashboard (which calls `cancelAll()`), there’s an “add more files“ screen, but I can’t add more files, since `allowNewUpload` is still `false`.

Same when I remove the file that’s being uploaded. Is this intended?

I think we should probably deprecate `reset()` in favor of `cancelAll()`, since they are basically the same thing?